_po/ja/how-to/laravel.po

```diff
@@ -639,33 +972,45 @@ msgid ""
 "</x-guest-layout>\n"
 "```"
 msgstr ""
+"                </div>\n"
+"            </div>\n"
+"        </div>\n"
+"    </div>\n"
+"</x-guest-layout>\n"
+"```"
 
 msgid "### Factory"
-msgstr ""
+msgstr "### ファクトリー"
 
 msgid ""
 "To create dummy data for this application, you need to create a Factory for Po"
 "st Model.\n"
 "Run following command which create a `database/factories/PostFactory.php` file"
 "."
 msgstr ""
+"Postモデル用のダミーデータを生成するファクトリーを作成します。"
+"次のコマンドを実行し`database/factories/PostFactory.php`ファイルを作成します。"
 
 msgid ""
 "```shell\n"
 "php artisan make:factory PostFactory\n"
 "```"
 msgstr ""
+"```shell\n"
+"php artisan make:factory PostFactory\n"
+"```"
 
 msgid ""
 "Let’s add some code to generate a fake data for Post Model. Here are before an"
 "d after:"
 msgstr ""
+"Postモデル用のダミーデータを生成するコードを書きます。変更前と変更後になります:"
 
 msgid "namespace Database\\Factories;"
-msgstr ""
+msgstr "namespace Database\\Factories;"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.